lib/datasource/go/releases-goproxy.ts

```diff
@@ -187,8 +175,22 @@ export async function getReleases({
 
   let result: ReleaseResult | null = null;
 
+  if (noproxy?.test(lookupName)) {
+    logger.debug(`Fetching ${lookupName} via GONOPROXY match`);
+    result = await direct.getReleases(config);
+    await packageCache.set(cacheNamespaces, cacheKey, result, cacheMinutes);
+    return result;
+  }
+
   for (const { url, fallback } of proxyList) {
     try {
+      if (url === 'off') {
+        break;
+      } else if (url === 'direct') {
+        result = await direct.getReleases(config);
```

Comment:
`direct.getReleases` throws for 404, in this case it would fallback to the next url (if any).
However, should we fallback when `null` was received? Proposed implementation means we shouldn't.

Reply:
🤷‍♂️

Reply:
Let's do this way and will see how it works for users